### DISCUSSION TITLE
Decrease the kernel log level by one notch?

### DISCUSSION THREAD

**Discussion by Pjotr123** [linuxmint/discussions#498] [Ideas]
Sometimes you get to see warnings when you boot your system. These are usually quite unimportant; Linux is notoriously talkative about even totally innocent "errors". Which is needlessly alarming for users.

Thankfully you can filter out the irrelevant warnings, leaving only the more important ones. Namely by decreasing the log level, which is set one notch too high in Linux Mint and Ubuntu.

The log level can be set from least verbose to most verbose. Namely to 0 (acute emergency), 1 (red alert), 2 (critical), 3 (error), 4 (warning), 5 (notice), 6 (info), or 7 (debug).

The default log level is 4. But for ordinary users, level 3 makes much more sense.

The fix is easy to apply: just add loglevel=3 to GRUB_CMDLINE_LINUX_DEFAULT in /etc/default/grub. Please do so for Mint 22.

**Comment by Jopp-gh** [comment]
Information is a right, in comparison, commercial Os's are much less open and transparent.

So, I'd rather prefer a preference in Settings or some diagnose tool to change the logging priority.

**Comment by josiebytes** [comment]
I agree with this. These errors are confusing for many non technical users. I think the log level should be set to 3 but like Jopp-gh said there should be a way to change this in settings.
